packages/teams-js/test/utils.ts

```diff
@@ -39,11 +39,14 @@ export class Utils {
   public parentWindow: Window;
   public topWindow: Window;
 
+  public respondWithTimestamp: boolean;
+
   private onMessageSent: null | ((messageRequest: MessageRequest) => void) = null;
 
   public constructor() {
     this.messages = [];
     this.childMessages = [];
+    this.respondWithTimestamp = false;
```

Comment:
who sets this to true?

Reply:
I set is as default by `false` to not add any possible issue with current tests that already ran. On the tests I made for testing the handler I set it to `true` to see if the handler gets called.